### PR TITLE
[Doc] Update data message limit in the API Overview guide

### PR DESCRIPTION
**Pull request by ltrung**
**Issue #1571 :**

**Description of changes:**
Update data message limit in the API Overview guide to clarify the hard limit per attendee and per meeting.
 
**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Looked at preview doc.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/03_API_Overview.md [hunk 1]
```diff
@@ -357,6 +357,10 @@ To receive messages on a given topic, set up a handler using the meetingSession.
 
 To unsubscribe the receive message handler, call meetingSession.audioVideo.[realtimeUnsubscribeFromReceiveDataMessage()](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#realtimeunsubscribefromreceivedatamessage).
 
-If you send too many messages at once, your messages may be returned to you with the [throttled](https://aws.github.io/amazon-chime-sdk-js/classes/datamessage.html#throttled) flag set. The current throttling soft limit for Data Messages is Rate: 100, Burst: 200. If you continue to exceed the throttle limit (hard limit: Rate: 500, Burst: 10000), then the server may hang up the connection.
+If you send too many messages at once, your messages may be returned to you with the [throttled](https://aws.github.
+io/amazon-chime-sdk-js/classes/datamessage.html#throttled) flag set. The current throttling soft limit for Data 
+Messages is Rate: 100, Burst: 200 for a meeting session. If you continue to exceed the throttle limit, then the
```
Comment (hensmi-amazon) [inline guides/03_API_Overview.md:362]: Can you define `Rate` and `Burst` ? whats the time window?

Comment (hensmi-amazon) [reply guides/03_API_Overview.md:362]: Also 'is  Rate: 100, Burst: 200' is a bit clunky.  Can we just describe the limits in plain english? i.e. 'is 100 per second or 200 for a burst of one minute' (not exactly like that, since the limit is not clear to me right now :P)

Comment (hensmi-amazon) [reply guides/03_API_Overview.md:362]: should we just say `meeting`? to me i feel like `meeting session` sounds like it could be an individual connection.

Comment (ltrung) [reply guides/03_API_Overview.md]: Thanks for your feedback. Updated.

**Review by hensmi-amazon — APPROVED** [review]
lgtm!
